catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/SparkServlet.java

```diff
@@ -129,6 +139,21 @@ public void init(ServletConfig config) throws ServletException {
                 false);
     }
 
+    private void registerProximityFactory() {
```

Comment:
Should this method make sure that the proximity function factory wasn't already registered?